### PR TITLE
Add grep flags for lines before, after the match

### PR DESCRIPTION
**Pull request by rahulkavale**

### CODE REVIEW

**Comment by agnivade** [comment]
I am kind of neutral on this change. Will wait for others to comment.

**Comment by waldyrious** [comment]
I'd rather say "use -A instead" and the rest could stay the same.

File: pages/common/grep.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 `grep -E {{^regex$}} {{path/to/file}}`
 
-- Print 3 lines of context around each match:
+- Print 3 lines of context around each match (supports -A for lines after the match, -B before the match):
```
Comment (ostera) [inline pages/common/grep.md]: maybe something like this?

```markdown
- Print 3 lines of [C]ontext around, [A]fter, or [B]efore each match:

`grep -{{C|A|B}} 3 {{search_string}} {{path/to/file}}`
```

Comment (waldyrious) [inline pages/common/grep.md]: Nice! I like this :)

**Comment by rahulkavale** [comment]
Makese sense, will update the PR

**Comment by waldyrious** [comment]
LGTM. Given the other maintainers' comments, I believe it's safe to assume an OK for merging. Thanks!

**Pull request merged**